docs/2021/ui/installation.md

```diff
@@ -99,3 +99,26 @@ npm start
 ```
 The React application will start on port 3000.
 Go to: http://localhost:3000
+
+**OR**
+
+using Docker
+
+FOSSology comes with a Dockerfile allowing the containerized execution.
+
+Run the following commands inside the project directory.
+
+```sh
+docker build \
+-t fossologyui:react1.0 \
+--build-arg REACT_APP_SERVER_URL="localhost/repo/api/v1" \
+--build-arg REACT_APP_HTTPS="false" .
+```
+
+```sh
+docker run -p 3000:3000 fossologyui:react1.0
+```
+
+The docker image can then be used using http://IP_OF_DOCKER_HOST:3000/ user fossy password fossy.
```

Comment:
A better working will be "The UI can then be accessed at http://IP_OF_DOCKER_HOST:3000/, user fossy password fossy."

What do you think?

Reply:
Yes, I also think that this is better and I have updated it.